OWNERS

```diff
@@ -4,7 +4,6 @@ approvers:
 reviewers:
   - ArangoGutierrez
   - cep235
-  - Deepthidharwar
```

Comment:
Would it make sense also to rm from https://github.com/openshift/release/blob/master/ci-operator/config/openshift/cluster-node-tuning-operator/OWNERS#L11 ?

Reply:
Thanks for looking Eduardo.  It would make sense, but if you read the first two lines in that file, it says the file is autogenerated and that we should not edit it.  It is autogenerated presumably from the OWNERS files in this repo.  So I believe once this PR merges, that file gets updated too.